app/Http/Controllers/Users/WorklogController.php

```diff
@@ -0,0 +1,88 @@
+<?php
```

Comment:
if this file has no use we can remove this for now

Reply:
Just using it for to render the dashboard view